### ISSUE TITLE
Make an official version of FormU similar to get80.sh

### ISSUE THREAD

**Issue by Melcore79** [StereoPhotoClinometry/Tools#1]
At some point we need to make an official tool to calculate FormU. Use get80.sh as a starting point.

BTW, also need to do a test in a directory to see if the value at the end of RESIDUALS.TXT matches FormU. Eric thinks it should, and John thinks the value at the end of RESIDUALS.TXT includes overlaps, whereas FormU does not include overlaps.

**Comment by terikdaly** [comment]
I know that the value at the end of RESIDUALS.TXT includes overlaps because
the value changes when I add or remove overlaps to LMKs. (e.g., if I veto a
bunch of overlaps by running veto.txt through lithos, the RMS residual will
go down).

**Comment by Melcore79** [comment]
TLDR: Looking at the code it does appear that overlaps are used to calculate the value at the end of RESIDUALS.TXT.

Longer version discussing more than just overlaps:

I confirmed that I could not recreate the value at the end of RESIDUALS.TXT using the output of get80.sh. get80.sh gives the RMS of pointing, limbs, overlaps, pointing + limbs, and pointing+limbs+overlaps for each landmark.

Eric looked through the code for RESIDUALS and it appears that the value at the end of RESIDUALS.TXT is an RMS of pointing, limbs, and overlaps for all landmarks. In other words, it is not an RMS of an RMS (i.e. something similar to FormU which weights each landmark equally), but rather a straight RMS. Which means if you have half the landmarks with 100 images and half with 3 images, the value at the end of RESIDUALS.TXT is going to heavily reflect the landmarks with 100 images since they have ~30 times the number of entries. But overlaps, as well as pointing and limbs, are included in that value.
